docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -547,8 +699,7 @@ kind: OpenStackDataPlaneDeployment
 metadata:
   name: openstack-pre-adoption
 spec:
-  nodeSets:
-  - openstack
+  nodeSets: $NODESETS
```

Comment:
In general this will work. There is a potential security hardening opportunity. As we deploy all the nodesets across multiple cells with the same deployment the ssh_known_hosts file on each node will have all the other nodes accross multiple cells. Even though no nova operation does cross cell compute to compute data transfer (only intra cell) 
I think this is fine as is now. But maybe we need a Note that separate deployment CR per cell also OK.